tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -9,39 +9,22 @@ load "${BATS_TEST_DIRNAME}/lib.sh"
 load "${BATS_TEST_DIRNAME}/confidential_common.sh"
 
 setup() {
-    if [ "${KATA_HYPERVISOR}" = "qemu-tdx" ]; then
-	    skip "${KATA_HYPERVISOR} is already running all the tests with guest-pulling, skip this specific one"
-    fi
-
-    if is_confidential_hardware; then
-        skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
-    fi
-
     if ! is_confidential_runtime_class; then
         skip "Test not supported for ${KATA_HYPERVISOR}."
     fi
 
     [ "${SNAPSHOTTER:-}" = "nydus" ] || skip "None snapshotter was found but this test requires one"
 
     setup_common
-    unencrypted_image_1="quay.io/sjenning/nginx:1.15-alpine"
-    unencrypted_image_2="quay.io/prometheus/busybox:latest"
+    unencrypted_image="quay.io/prometheus/busybox:latest"
     large_image="quay.io/confidential-containers/test-images:largeimage"
 }
 
 @test "Test we can pull an unencrypted image outside the guest with runc and then inside the guest successfully" {
-    if is_confidential_hardware; then
-        skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
-    fi
-
-    if ! is_confidential_runtime_class; then
-        skip "Test not supported for ${KATA_HYPERVISOR}."
-    fi
```

Comment:
I think we still need this check so that these test don't run on non coco runtimeclasses e.g. qemu-stratovirt?

Reply:
The check will run in both `setup()` and `teardown()` for each test. From my understanding, if the runtime class is not tee’s, should it be skipped directly in the `setup()` phase? Please correct me if I have misunderstood anything.

Reply:
Ah sorry, I missed that this was in the test and we also had the setup and teardown. My mistake!